apps/tasks/signals.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import django.dispatch
+from django.dispatch import receiver
+
+logger = logging.getLogger(__name__)
+
+ALL = ['task_status_changed', 'task_manager_request_status_update']
+
+task_status_changed = django.dispatch.Signal(providing_args=['instance', ])
+celery_request_status_update = django.dispatch.Signal(providing_args=['task_id', 'job_id', 'status'])
+
+
+@receiver(celery_request_status_update)
+def on_task_manager_request_status_update(sender, task_id, job_id, status, **kwargs):
+    logger.debug('on_task_manager_request_status_update {} {}'.format(job_id, status))
+
+    from tasks.models import Task, TaskStatus
```

Comment:
Shouldn't we use REST API for this? Theoretically task runner can be on separate machine.